src/wallet/rpcwallet.cpp

```diff
@@ -2272,7 +2272,7 @@ UniValue getwalletinfo(const UniValue& params, bool fHelp)
             "\nResult:\n"
             "{\n"
             "  \"walletversion\": xxxxx,     (numeric) the wallet version\n"
-            "  \"balance\": xxxxxxx,         (numeric) the total confirmed balance of the wallet in " + CURRENCY_UNIT + "\n"
```

Comment:
This is a bit weird; surely the confirmed balance would be more useful? (Rather than having to get that by subtracting unconfirmed from confirmed+unconfirmed.)

Reply:
It's also just the transparent balance so that might be worth adding to avoid confusion.

Reply:
Ugh that's (even more) terrible. Can't we just fix this? (We would have to check compatibility with existing wallets if we change the semantics.)

Reply:
I double-checked on testnet and the code again.  It is just the confirmed and spendable transparent balance.